tests/storage/cross_cluster_live_migration/test_cclm.py

```
@@ -85,3 +87,49 @@ def test_compute_live_migrate_vms_after_cclm(self, local_vms_after_cclm_migratio
     @pytest.mark.polarion("CNV-14334")
     def test_source_vms_can_be_deleted(self, vms_for_cclm):
         assert_vms_can_be_deleted(vms=vms_for_cclm)
+
+    @pytest.mark.polarion("CNV-15237")
+    def test_target_vms_can_be_deleted(self, local_vms_after_cclm_migration):
+        assert_vms_can_be_deleted(vms=local_vms_after_cclm_migration)
+
+
+@pytest.mark.parametrize(
+    "dv_wait_timeout, vms_for_cclm",
```

Ruff (0.15.7)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)